TheWarWithin/DemonHunterHavoc.lua

```diff
@@ -582,18 +616,18 @@ spec:RegisterAuras( {
     -- https://wowhead.com/beta/spell=162264
     metamorphosis = {
         id = 162264,
-        duration = function () return 24 + ( pvptalent.demonic_origins.enabled and -15 or 0 ) end,
+        duration = 20,
         max_stack = 1,
-        meta = {
+        --[[meta = {
             extended_by_demonic = function ()
                 return false -- disabled in 8.0:  talent.demonic.enabled and ( buff.metamorphosis.up and buff.metamorphosis.duration % 15 > 0 and buff.metamorphosis.duration > ( action.eye_beam.cast + 8 ) )
             end,
-        },
+        },--]]
     },
-    momentum = {
+    exergy = {
```

Comment:
Fair chance this will still/also be referrred to as `momentum` in SimC; should make a `copy` so both names are supported.